scalafix-cli/src/main/scala/scalafix/internal/v1/MainOps.scala

```diff
@@ -267,7 +269,6 @@ object MainOps {
           )
         } catch {
           case NonFatal(_) =>
-            args.global.restart()
```

Comment:
is this related to the change?

Reply:
no, but I don't understand why we do that. 
even if the compiler crash, the global don't crash. We can still close it after this.

Reply:
this what happens if we let this line
```try {
          InteractiveSemanticdb.toTextDocument(
            g,
            doc.input.text,
            doc.internal.input.syntax,
            10000,
            Nil
          )
          throw new Exception("exception")
        } catch {
          case NonFatal(_) =>
            println("exception catched")
            args.global.restart()
            TextDocument.defaultInstance
        }
        ```
![image](https://user-images.githubusercontent.com/7843237/109974089-7cf40580-7cf9-11eb-9952-35938cce4e96.png)

Reply:
it's never closed

Reply:
I see, I just realized`restart()` was just dropping the reference https://github.com/scalacenter/scalafix/blob/e31e5c946a14bb3839ef5c087b660953a3485db8/scalafix-core/src/main/scala/scalafix/internal/v1/LazyValue.scala#L17-L19 :+1: